src/Elliptic/Actions/InitializeBackgroundFields.hpp

```diff
@@ -0,0 +1,75 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <tuple>
+#include <utility>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Domain/Tags.hpp"
+#include "ParallelAlgorithms/Initialization/MutateAssign.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+namespace Parallel {
+template <typename Metavariables>
+struct GlobalCache;
+}  // namespace Parallel
+/// \endcond
+
+namespace elliptic::Actions {
+
+/*!
+ * \brief Initialize the variable-independent background fields for an elliptic
```

Comment:
[optional] It's not clear to me what you mean by a "variable-independent background field". Likely this is my fault for being unfamiliar with details of elliptic solvers, in which case I'd be happy with you just clarifying for me in a Github comment and leaving the code as-is. But if you think it'd be valuable to clarify in the Doxygen comment, I won't object ;)

Reply:
Add this example to the documentation:

> Examples for background fields would be a background metric, associated curvature quantities, or matter sources such as a mass-density in the XCTS equations.